packages/core/scripts/local/DeployEMP.js

```diff
@@ -21,12 +21,13 @@
  *
  * Example: yarn truffle exec ./packages/core/scripts/local/DeployEMP.js --network test --test true --identifier ETH/BTC --cversion 1.1.0
```

Comment:
You could drop the `--cversion` flag in the example or set it to the default version of `1.2.2`.